packages/EasyRandom/readme.md

```diff
@@ -71,6 +71,16 @@ $randomGenerator = (new \EonX\EasyRandom\RandomGenerator())->setUuidV4Generator(
 $uuidV4 = $randomGenerator->uuidV4();
 ```
 
+#### In Symfony configure your preferred UUID V4 generator
```

Comment:
Here can you please simply explain that this is possible because by default the bundle registers the built-in generators as services.